### DISCUSSION TITLE
Read-only and default input in Default theme look the same

### DISCUSSION THREAD

**Discussion by pguptaDS** [twilio-labs/paste#1015] [Q&A] [answered]
The read-only state and the default state for the Input component (Default theme) look very similar in the sketch library (see screenshot below). I cross-checked with the Paste website and there is definitely more visual difference for the read-only state. Is this something that the team is aware of? 
![image](https://user-images.githubusercontent.com/64559618/101557476-f043e080-3971-11eb-9842-528c3f8332e9.png)

**Comment by serifluous** [comment, marked as answer]
Hey @pguptaDS ! If you inspect the browser and Sketch, the two should give you the same hex or rgb values. The visual difference might be because Sketch and the browser display colors differently (not sure why—design tooling is sometimes weird 😞). If you aren't seeing the same values, let us know. 

_However_ you're also not the first person to give feedback about how the read-only state is too similar to the default state, so that's something we definitely want to improve with the form fields regardless!
